Backend/CourseAppUserService/CourseAppUserService_Application/Common/Exceptions/UserValidationException.cs

```diff
@@ -0,0 +1,19 @@
+using FluentValidation;
+using FluentValidation.Results;
+
+namespace CourseAppUserService_Application.Common.Exceptions;
+
+public class UserValidationException: ValidationException
+{
+    public List<string> errors { get; set; }
```

Comment:
Properties names in CamelCase
```suggestion
    public List<string> Errors { get; set; }
```